apps/perf-test/tasks/flamegraph/viz.js

```diff
@@ -0,0 +1,303 @@
+'use strict';
```

Comment:
Is this basically a modified copy of the existing flamebearer code?

Reply:
Yep.